compoments/reportRepair.js

```diff
@@ -10,80 +10,89 @@ import Communal from './reportRepair/communal';
 import NotEligibleCommunalRepairs from './reportRepair/NotEligibleCommunalRepairs'
 import RepairLocation from './reportRepair/repairLocation';
 
-import { BackLink } from 'govuk-react';
+import BackLink from "./backLink";
 import Flow from '../flow';
-import {
-  Redirect,
-  Switch,
-  Route,
-  useHistory,
-  useRouteMatch
-} from 'react-router-dom';
+
+import Router from 'next/router'
 
 export default function Report() {
-  let history = useHistory();
-  let { path, url } = useRouteMatch();
+
+  let thing = 'thing'
+  console.log('bwooo')
+  // let history = useHistory();
```

Comment:
Commented out code (here and below) - are they needed? ... is this a work in progress?